chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -960,6 +960,14 @@ endif::VK_KHR_display_swapchain[]
     previously submitted for execution.
 ****
 
+fname:vkQueuePresentKHR is a <<devsandqueues-submission,queue submission
+command>> that has a single _batch_ of work: submitting a presentation
```

Comment:
Hm, there should be something about when that batch finishes w.r.t. `vkAcquire`. At least, one would assume when all the presented s. images are acquired after, then the batch is finished (and the semaphore can be destroyed) too in such case.